tests/verify_membackend.erl

```diff
@@ -152,7 +152,10 @@ check_put_delete(Node) ->
     Mem = get_used_space(Pid, Node),
 
     %% this is meh, but the value isn't always the same length.
-    case (Mem == MemBaseline - 1142) orelse 
+    %% It seems to be the size of a Riak Object
+    case (Mem == MemBaseline - 1146) orelse
```

Comment:
Yeah.  Well. What is the point of checking this anyway? To make sure there's not some kind of memory leak?